src/ScheduleService.Business/Category/CreateCategoryCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.ScheduleService.Business.Category.Interfaces;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Category;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Business.Category;
+
+public class CreateCategoryCommand : ICreateCategoryCommand
+{
+  public Task<IActionResult> ExecuteAsync(CreateCategoryRequest request)
```

Comment:
For now we won't use IActionResult. Use OperationResultResponse<T>. T will be your return type